jp/components/grid_search.md

```diff
@@ -74,16 +74,16 @@ public exactMatch: boolean = false;
 
 #### 検索入力ボックス
 
-Now let's create our search input! By binding our **searchText** as ngModel to our newly created input and subscribe to the ngModelChange event, we can detect every single **searchText** modification by the user. This will allow us to use the grid's `findNext` and `findPrev` methods to highlight all the occurrences of the **searchText** and scroll to the next/previous one (depending on which method we have invoked).
+検索入力を作成します。**searchText** を ngModel として新しく作成した入力へバインドして ngModelChange イベントにサブスクライブします。ユーザーによるすべての ngModelChange 変更を検出できます。これによってグリッドの `findNext` と `findPrev` メソッドを使用して searchText のすべての出現を強調し、次へまたは前 (呼び出すメソッドに基づいて) へスクロールできます。
```

Comment:
The second instance of "ngModelChange" should be "searchText".  Also, `searchText` should be `**searchText**`.